code/datums/gamemode/role/catbeast.dm

```diff
@@ -46,7 +46,7 @@ var/list/catbeast_names = list("Meowth","Fluffy","Subject 246","Experiment 35a",
 	D1.origin = "Loose Catbeast"
 	D1.makerandom(str, rob, anti, bad)
 	H.infect_disease2(D1, 1, "Loose Catbeast")
-	antag.store_memory(D1.get_info())
+	antag.store_memory(D1.get_info(), 1)
```

Comment:
you can write `(D!.get_info(), forced = TRUE)` to make it more obvious what this is doing

not require but it makes this nicer to look at